### DISCUSSION TITLE
Include MongoDB ObjectId ↔ Timestamp Converter

### DISCUSSION THREAD

**Discussion by juanmorenocttexpress** [CorentinTh/it-tools#789] [Ideas]
Hi all, it would be nice to have this tool included. It makes search data in mongo in a range of dates easy:

https://steveridout.com/mongo-object-time/

**Comment by sharevb** [comment]
Hi @juanmorenocttexpress and @CorentinTh , implemented in #1233

**Comment by sharevb** [comment]
Hi, if you arr interested in up to date version of it-tools, I made kind of a fork here : https://github.com/sharevb/it-tools (https://sharevb-it-tools.vercel.app/ and docker images https://github.com/sharevb/it-tools/pkgs/container/it-tools)
